daemon/process/settings/dobby.llama.json

```diff
@@ -0,0 +1,34 @@
+{
+  "dbus": {
+    "serviceName": "com.sky.dobby",
```

Comment:
This is still `com.sky.dobby` (and other sky paths in this settings file need changing)

Reply:
Nice catch, I missed this file as it was deleted when I was making all sky->rdk change, and now when it got reverted I missed it completely.